lib/ReactCrop.js

```diff
@@ -109,22 +109,25 @@ function resolveCrop(crop, image) {
   return crop;
 }
 
-function getPixelCrop(image, percentCrop) {
+function getPixelCrop(image, percentCrop, useNaturalImageDimensions) {
```

Comment:
Plz change this to `(image, percentCrop, useNaturalImageDimensions = true)`, since this function is exported people could be depending on the default behaviour